### ISSUE TITLE
Bring Your Own Certificate Authority

### ISSUE THREAD

**Issue by tkuester** [tkuester/taky#24]
The tools to build the server (and client) certificates should allow you to use your own CA. While it doesn't matter in most use cases, some users may have an existing CA infrastructure setup on their EUDs, and may wish to use that CA.

**Comment by pinztrek** [comment]
Agreed this is important. I've been looking into if appropriate files could be dropped into location and used. 
Tnx, Alan ("Pinzgauer" on discord)
